src/graphics/program-lib/programs/basic.js

```diff
@@ -133,15 +133,16 @@ var basic = {
             code += "   alphaTest(gl_FragColor.a);\n";
         }
 
-        if (options.pass === SHADER_PICK) {
+        if (options.pass !== SHADER_PICK) {

```

Comment:
If this `if` statement is **not** a pick pass, how come the next comment is `PICK PASS`? 😄

Reply:
I will remove it

Reply:
removed